belt_nlp/bert_with_pooling.py

```diff
@@ -75,7 +78,7 @@ def __init__(
         self._params.update(additional_params)
 
         self.device = device
-        self.collate_fn = BertClassifierWithPooling.collate_fn_pooled_tokens
+        self.collate_fn = BertBaseWithPooling.collate_fn_pooled_tokens
```

Comment:
But this is is a static method, it seems natural to me to use with the class and not an instance.

Reply:
You may be right in general, but this is a specific case, where the `collate_fn_pooled_tokens` method is called in an instance method of the same class.

If you create a new class that derives from the `BertBaseWithPooling` class and overwrite the `collate_fn_pooled_tokens` method, you would probably expect the overwritten method to be used in the `__init__` method and that's why I think it would be safer to call it as `self.collate_fn_pooled_tokens`.

However, in this very specific case, this scenario may be unlikely, so I just want to present you my point of view, but I leave the decision to you.

Reply:
Thanks for clarification. Yes, you are right, I will change it.